src/github/mod.rs

```diff
@@ -579,6 +586,7 @@ fn calculate_permission_diffs(
 /// Returns `None` if the bot is not an actual bot user, but rather a GitHub app.
 fn bot_user_name(bot: &Bot) -> Option<&str> {
     match bot {
+        // FIXME: set this to `None` once homu is removed completely
```

Comment:
can you track this in an issue? 👍

Reply:
I thought you'd ask this :laughing: Good idea. https://github.com/rust-lang/sync-team/issues/95